src/pages/Article/ArticlePage.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PageLayout from '../../components/PageLayout';
+import Button from "@mui/material/Button";
+import { Box } from '@mui/system';
+
+/**
+ * Renders the Article page.
+ * @returns {JSX.Element} ArticlePage component
+ */
+function ArticlePage({article}) {
+	
+	const articleDemo = {
+		title: "What is Coding?",
+		content: `Coding is the practice of building websites and apps by writing out instructions in a programming language. 
+		It governs how data is collected, how sites appear to the user, and every aspect of how you engage with the web. 
+		The background on this page appears the way it does due to lines of code. And when you enter your email address in a pop-up box, code is guiding the way your email is stored and shared
+		Computers can perform extremely fast calculations, complete complex mathematical tasks, and store large amounts of data. 
+		use binary code to communicate and operate.Computer programming (or simply coding) refers to the task of writing  instructions that, once compiled, computers will understand and follow.  
+		It therefore allows us to create a range of different software from operating systems and business packages to mobile apps and video games.  
+		It therefore allows us to create a range of different software from operating systems and business packages to mobile apps and video games.Machines use binary code to communicate and operate.  
+		Computer programming (or simply coding) refers to the task of writing  instructions that, once compiled, computers will understand and follow.  
+		It therefore allows us to create a range of different software from operating systems and business packages to mobile apps and video games.  `,
+		link: "https://www.thinkful.com/blog/what-is-coding/"
+	};
+	
+	
+	const articleDisplay = article || articleDemo; {/* if no article is passed then the demo article object above is displayed */ }
+  
+	return (
+		<PageLayout viewportPage title='Article'>
+			<Box sx={{ 
+				display: "flex",
+				flexDirection: "column",
+				justifyContent: "center",
+				alignItems: "center",
+				height: "calc(100% - 60px)", 
+				overflowY: "auto"
+				}}>
+
+				<Box  sx={{height: "80%",width: "40%",display:"flex",justifyContent: "center",alignItems: "center",backgroundColor: "#D9D9D9", padding:"20px", overflowY:"auto" }}>
```

Comment:
width: 40% is not sufficient for a viewport width of 900.
<img width="942" alt="image" src="https://github.com/SE2-WS23/Skillverse-Web/assets/37250473/45aa9c1f-917d-43c0-a2ce-f65f8d83999c">
I think you can use a higher width or breakpoints

Reply:
I increased the width to 60% and it looks better for article display, but it different from what we have on the mock-up. 
check it and see how it looks.

Reply:
I still think, that this is a waste of place, if you use a small device. Maybe you consider to use a min-width. The Mock-up is (sadly) not our only point of truth, so sometimes we have to adapt something, so that is sufficient.

Reply:
<img width="829" alt="Screenshot 2023-11-13 at 10 06 28" src="https://github.com/SE2-WS23/Skillverse-Web/assets/104029083/c96b4556-862d-450a-8557-b86e54acdcbc">


how it looks for small device like iPad